src/server/node_manager.c

```diff
@@ -8693,7 +8701,12 @@ free_sister_vnodes(job *pjob, char *vnodelist, char *err_msg,
 	/* increment everything found in new exec_vnode */
 	set_resc_assigned((void *)pjob, 0,  INCR);
 						
-	set_scheduler_flag(SCH_SCHEDULE_TERM);
+	if (find_assoc_sched_pj(pjob, &psched))
+		set_scheduler_flag(SCH_SCHEDULE_TERM, psched);
+	else {
+		sprintf(log_buffer, "Unable to reach scheduler associated with job %s", pjob->ji_qs.ji_jobid);
```

Comment:
Please update it to "Unable to find scheduler associated with partition"

Reply:
Done